parser/src/main/java/io/specmesh/apiparser/model/SchemaInfo.java

```diff
@@ -14,6 +17,20 @@ public class SchemaInfo {
     private final String contentType;
     private String schemaLookupStrategy;
 
+    /**
+     * @param schemaRef
+     *            location of schema
+     * @param schemaFormat
+     *            format of schema
+     * @param schemaIdLocation
+     *            ???
+     * @param schemaIdPayloadLocation
+     *            ???
```

Comment:
similar to above - a confluent serde assumes it - examples here: https://gist.github.com/dalelane/3931c17b14c51fa4a1cf25496237d188